raven/processes/wps_nalcms_zonal_stats.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import json
+import tempfile
+
+from pywps import LiteralInput, ComplexInput
+from pywps import ComplexOutput
+from pywps import Process, FORMATS
+from pywps.app.Common import Metadata
+from rasterstats import zonal_stats
+
+from raven.utils import archive_sniffer, crs_sniffer, generic_vector_reproject, single_file_check
+
+LOGGER = logging.getLogger("PYWPS")
+
+NALCMS_CATEGORIES = {
```

Comment:
This is just style, but I think it would be cleaner to do 
```
categories = {'Forest':[1,2,3,4,5,6], 'Shrubs':[7,8,11], ...} 
NALCMS_CATEGORIES = {i:cat for (cat, ids) in categories.items() for i in ids}
```

Reply:
Interesting. Sure.